docs/explanations/faq.rst

```diff
@@ -1,23 +1,68 @@
+Troubleshooting and FAQ
+=======================
+
+Please visit the `IDAES discussions board on Github <https://github.com/IDAES/idaes-pse/discussions>`_ for more questions and answers.
+
+.. contents::
+    :depth: 2
+    :local:
+
 Frequently Asked Questions
-==========================
+--------------------------
+
+Help! My model does not solve!
+``````````````````````````````
+
+The IDAES team does not have a lot of resources for supporting users to solve their models, so before contacting the IDAES team for help, make sure you have tried all the steps below.
```

Comment:
I think I would prefer `The IDAES Team has limited resources and bandwidth for providing direct support for assisting users in debugging solver issues.  Before contacting...`